cloudpickle/cloudpickle.py

```diff
@@ -78,6 +78,10 @@
     PY3 = True
 
 
+# cache that tracks the values of global variables.
+_BASE_GLOBALS_CACHE = {}
```

Comment:
I am worried that this module level dict will cause memory leaks if loky is used in long-running processes with dynamic modules. Is this cache really necessary?